tests/test_aside.py

```diff
@@ -166,17 +190,28 @@ def test_responses(self):
             course_key=course_id,
             problem_usage_key=UsageKey.from_string(unicode(problem_id) + "extra"),
             user_id=user.id,
-            answer_id='answer_0',
-            answer_text='Answer #0',
+            answer_id=answer_data[0]['answer_id'],
+            answer_text=answer_data[0]['answer_text'],
             event={}
         )
 
-        resp = self.aside_instance.responses()
+        request = RequestFactory().request()
+        request.user = self.instructor
+        request.session = request.environ
+        store = modulestore()
+        problem = store.get_item(problem_id)
+        problem.runtime = self.runtime
+        problem.xmodule_runtime = self.runtime
+        aside_block = get_aside_from_xblock(problem, self.aside_usage_key.aside_type)
```

Comment:
Could/should we move this block (up to line 198) to the top of the test case under the assignment for `problem_id`? It seems appropriate to group together the code that creates (a) test data, and (b) variables/objects that you'll need to interact with